tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc

```diff
@@ -61,6 +62,20 @@ Status SetName(HloModule *module, HloInstruction *gemm) {
   return OkStatus();
 }
 
+// Returns whether a given PrimitiveType is supported by cuBLASLt Epilogue
+// Fusion.
+bool SupportsEpilogueFusion(PrimitiveType type) {
```

Comment:
Can we cite where in the cublas docs it says that int8 doesn't support epilogue fusion?  I'm not seeing it.

(Or is this written this way because we don't currently support int8 in cublasLt?  If that's the case, please add a comment to this effect.)

Reply:
This table provides the data types supported by Epilogue Fusion: https://docs.nvidia.com/cuda/cublas/index.html#cublasLtMatmul

Reply:
In the table, all rows with "Non-default epilogue not supported" in the final column correspond to data types for which Epilogue Fusion (i.e. a non-default Epilogue) is not supported. For instance, the third row represents GEMMs based on INT8 input and output matrices and shows that Epilogue Fusion is not available in this case. The following row indicates that BF16 is supported. I'll add this information in a comment.